app/views/layouts/application.html.slim

```diff
@@ -3,7 +3,11 @@ html
   head
     title Voluntaries Platform
     = stylesheet_link_tag 'application', media: 'all',
-    | 'data-turbolinks-track': 'reload'
+      'data-turbolinks-track': 'reload'
```

Comment:
right, slim used to be lint, and my editor still does, so thats why I'm mistaken